tests/test_meshtastic_connection.py

```
@@ -564,6 +564,55 @@ def _configure_ble(self) -> str:
         mu.meshtastic_iface = None
         return ble_address
 
+    def test_interface_creation_shutdown_is_not_reported_as_timeout(self, monkeypatch):
```

Ruff (0.16.0)

[warning] Unused method argument: `monkeypatch`

(ARG002)